### ISSUE TITLE
Get-D365InstalledHotfix broken

### ISSUE THREAD

**Issue by skaue** [d365collaborative/d365fo.tools#134]
This is private hosted cloud VM with local admin user. The cmdlet fails. I will see if I have time to look into it.

![image](https://user-images.githubusercontent.com/14696/47707468-2c050b80-dc2c-11e8-9ae2-7f27fe177f46.png)

**Comment by Splaxi** [comment]
I have some time at the airport later today. I'll give it a look as well ;)

**Comment by Splaxi** [comment]
Please update the cmdlet with this, starting on line 97

```
$files = @((Join-Path -Path $BinDir -ChildPath "Microsoft.Dynamics.AX.Metadata.Storage.dll"),
            (Join-Path -Path $BinDir -ChildPath "Microsoft.Dynamics.ApplicationPlatform.XppServices.Instrumentation.dll"))
```

**Comment by skaue** [comment]
Works. Thanks!

**Comment by Splaxi** [comment]
Will you or should I create the PR with the change? 😀

**Comment by skaue** [comment]
I created a PR.

**Comment by Splaxi** [comment]
Should be solved in 0.4.81
